pkg/capabilities/pb/capabilities_helpers.go

```diff
@@ -63,8 +63,11 @@ func CapabilityRequestToProto(req capabilities.CapabilityRequest) *CapabilityReq
 			ReferenceId:              req.Metadata.ReferenceID,
 			DecodedWorkflowName:      req.Metadata.DecodedWorkflowName,
 		},
-		Inputs: values.ProtoMap(inputs),
-		Config: values.ProtoMap(config),
+		Inputs:        values.ProtoMap(inputs),
```

Comment:
I'd add the same comment here -- we should know what type we're dealing with; either v1 or v2.